### PR TITLE
Add info on how to ask Qs about CC appts; remove mention of Chrome Remote Desktop

### PR DESCRIPTION
**Pull request by willfurnass**

### CODE REVIEW

**Review by annakrystalli — APPROVED** [review]
LGTM. Perhaps also being explicit that the autogenerated emails are not monitored. Ie this is at the top of every autogenerated email.
```
This is an auto-generated e-mail in response to submission of the code clinic booking form.
```

Might also be a good idea to explicitly add in the autogenerated emails not to reply (one for @twinkarma )

File: pages/support/code-clinic.md [hunk 1]
```diff
@@ -88,6 +84,11 @@ If you cannot attend the session,
 please give a **No** response on the calendar invite
 so we can proceed to remove your booking from the system.
 
+### Any questions before your session?
+
+In the first instance please contact the helper assigned to your request (see your Google Calendar appointment event for details)
+or if a helper hasn't been assigned yet please contact `code-clinic-helpers-group@sheffield.ac.uk`.
```
Comment (willfurnass) [inline pages/support/code-clinic.md:90]:
```suggestion
or if a helper hasn't been assigned yet please contact `code-clinic-helpers-group@sheffield.ac.uk`.

Please do not reply directly to auto-generated emails received after booking a Code Clinic session - the associated mailbox is not monitored.
```

**Review by willfurnass — COMMENTED** [review]
.

**Comment by willfurnass** [comment]
@annakrystalli Thanks.

@twinkarma It would be great if you could look at Anna's suggestion.

**Comment by willfurnass** [comment]
@twinkarma NB I've also added this info to the Google Form autoreply - does it need to be duplicated elsewhere?

**Pull request merged**